src/pages/dev/general-message-passing/solidity-utilities.mdx

```diff
@@ -1,33 +1,55 @@
 # Solidity utilities
 
-To facilitate cross-chain development, we have provided some Solidity utilites. These can be found [here](https://github.com/axelarnetwork/axelar-utils-solidity). Each of these is described below, along with their proposed use.
+To facilitate interchain development, we have provided some Solidity utilites. These can be found [here](https://github.com/axelarnetwork/axelar-utils-solidity). Each of these is described below, along with their proposed use.
```

Comment:
we should link to the contract deployment info files somewhere for the full set of addresses

Reply:
Sorry unclear on this. You mean the address of create2 and create3 addresses? Those addresses are included in the docs. Which contract deployment info file are you referring to? Thx!

Reply:
Following up on this @milapsheth just hope to clarify which file you mean here. Are you referencing this file's full set of addresses?
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/mainnet.json
Thanks